nxdrive/metrics/utils.py

```diff
@@ -0,0 +1,50 @@
+import platform
+from functools import lru_cache
+from typing import Tuple
+
+from .. import __version__
+from ..constants import APP_NAME, MAC, WINDOWS
+
+
+@lru_cache(maxsize=1)
+def _get_current_os_details() -> Tuple[str]:
+    """Get OS details: name, full version, simplified version [x.y].
+
+    Examples:
+        - ("macOS", "10.15.3", "10.15")
+        - ("Windows", "10.0.19041", "10.0")
```

Comment:
Is it enought to only have "Windows"? Should we use "Microsoft Windows" or even "Microsoft Windows NT"?

Reply:
Do we need stats about the different types of Windows ?

Reply:
I would say no.